libflux/flux-core/src/semantic/nodes.rs

```diff
@@ -130,6 +130,7 @@ pub enum Expression {
     Boolean(BooleanLit),
     DateTime(DateTimeLit),
     Regexp(RegexpLit),
+    Vector(Box<VectorExpr>),
```

Comment:
Again here, there is no need to add a new `Vector` expression to the semantic graph nodes.